addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/ZoneControl.java

```diff
@@ -0,0 +1,292 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.yamahareceiver.internal.protocol;
+
+import java.io.IOException;
+import java.lang.ref.WeakReference;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.xml.parsers.ParserConfigurationException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+/**
+ * The zone protocol class is used to control one zone of a Yamaha receiver with HTTP/xml.
+ * No state will be saved in here, but in {@link ZoneControl.State} instead.
+ *
+ * @author David Gräff <david.graeff@tu-dortmund.de>
+ * @author Eric Thill
+ * @author Ben Jones
+ * @since 2.0
```

Comment:
Is there a maven command to clear the author/version tags? In OH1 addons it was common to list authors and versions and I didn't find a documentation saying how annotations in the OH2 codebase should look like.